### PR TITLE
feat(explorer): selector to switch network

### PR DESCRIPTION
**Pull request by MarcosNicolau**
## Description

Adds a selector in the explorer nav to switch the network.

**Testing**
- Run the explorer locally and you should see the new selector:

**large screens**:
image

**smalls screens**:
image

## Type of change
- [x] New feature

## Checklist

- [ ] “Hotfix” to `testnet`, everything else to `staging`
- [ ] Linked to Github Issue
- [ ] This change depends on code or research by an external entity
  - [ ] Acknowledgements were updated to give credit
- [ ] Unit tests added
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [ ] This change is an Optimization
  - [ ] Benchmarks added/run
- [ ] Has a known issue
  - [Link to the open issue addressing it]() 
- [ ] If your PR changes the Operator compatibility (Ex: Upgrade prover versions)
  - [ ] This PR adds compatibility for operator for both versions and do not change batcher/docs/examples
  - [ ] This PR updates batcher and docs/examples to the newer version. This requires the operator are already updated to be compatible

### CODE REVIEW

File: explorer/lib/explorer_web/live/utils.ex [hunk 1]
```diff
@@ -108,6 +108,24 @@ defmodule ExplorerWeb.Helpers do
...
+    String.capitalize(prefix)
+  end
+
+  @doc """
+    Returns a list of available AlignedLayer networks with their names and explorer URLs.
+  """
+  def get_aligned_networks() do
+    [
+      {"Mainnet", "https://explorer.alignedlayer.com"},
+      {"Holesky", "https://holesky.explorer.alignedlayer.com"}
```
Comment (uri-99) [inline explorer/lib/explorer_web/live/utils.ex:125]: missing staging

**Review by uri-99 — APPROVED** [review]

**Review by uri-99 — CHANGES_REQUESTED** [review]
we should go to staging

**Comment by uri-99** [comment]
The many file changes should go away when #1625  is merged

**Review by uri-99 — APPROVED** [review]

**Review by PatStiles — APPROVED** [review]

**Review by Oppen — APPROVED** [review]
It works on my machine. It's a little odd that because it works by redirecting to the other domains the UI itself changes. I would have expected as user that only the data it showed would change.

**Comment by MarcosNicolau** [comment]
> It works on my machine. It's a little odd that because it works by redirecting to the other domains the UI itself changes. I would have expected as user that only the data it showed would change.

It also seemed odd to me at first, but if you check sites like [etherscan](https://etherscan.io/) they do the same.

**Pull request merged**